helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -342,6 +342,9 @@ receivers:
           com.splunk.sourcetype: 'EXPR("kube:container:"+$$.container_name)'
         attributes:
           log.iostream: 'EXPR($$.stream)'
+      {{- with .Values.logsCollection.containers.extraOperators }}
+      {{ . | toYaml | nindent 6 }}
+      {{- end }}
```

Comment:
Why don't put it after `clean-up-log-record`? I think it's better to apply custom operators on a record that user gets at the end of the default transformation. We don't lose any data after `clean-up-log-record`, right?

Reply:
that makes sense. updated.

Reply:
Thanks. I believe this is a breaking change, right? User's `extraOperators` configs have to be changed to adopt to the new log structure that we have at this point or they will work as before?

Reply:
Yes. 
Since we put it after clean up log record, it is breaking change if they were using `$$.pod_name` instead of `$$resource["k8s.pod.name"]`

it is not a breaking change if we just put it before multiline config as my original commit.

Reply:
I think it's still much cleaner to provide a way to manipulate records after all the default transformation is done on the whole recombined logs instead of on each broken line. If we keep it before multiline, user can break the further recombine operator.

Can you please mark this as a breaking change in Chandelog and add the steps on how to upgrade in https://github.com/signalfx/splunk-otel-collector-chart/blob/main/UPGRADING.md?